swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      version:
+        $ref: '#/definitions/version'
+      alerts:
+        type: array
+        items:
+          $ref: '#/definitions/Alert'
+      charts:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      correction_notice:
+        type: string
+        example: "A correction has been ma…gise for any inconvenience."
+      downloads:
+        $ref: '#/definitions/Downloads'
+      equations:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      headlines:
+        type: array
+        description: Headlines from this bulletin
+        maxItems: 3
+        items:
+          type: string
+          example: "Headline about this bulletin"
+      images:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      meta_description:
+        type: string
+        example: "Estimates of employment… statistics for the UK."
+      sections:
+        type: array
+        items:
+          $ref: '#/definitions/ContentSection'
+      summary:
+        $ref: '#/definitions/summary'
+      tables:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      update_date:
+        type: string
+        format: 'date-time'
+        example: "2019-11-12T15:53:59.810Z"
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          bulletin:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+
+  Alert:
+    type: object
+    properties:
+      date:
+        type: string
+        format: 'date-time'
+        example: "2019-09-29T23:00:00.000Z"
+      description:
+        type: string
+        description: Markdown formatted description of the alert
+        example: "The Business investment by industry and ...for the inconvenience"
+      type:
+        type: string
+        example: "alert"
+
+  Contact:
+    type: object
+    properties:
+      email:
+        type: string
+        example: "joe.bloggs@ons.gov.uk"
+      name:
+        type: string
+        example: "Joe Bloggs"
+      telephone:
+        type: string
+        example: "+44 (0)845 601 3034"
+
+  ContentSection:
     type: object
     properties:
-      message:
+      title:
         type: string
-        description: "Message returned by hello world endpoint"
-        example: "Hello, world!"
\ No newline at end of file
+        description: Title of the section
+        example: "Section Title"
+      content:
+        type: string
+        description: Markdown formatted content for the section
+        example: "Some _content_ in [Markdown](https://en.wikipedia.org/wiki/Markdown)"
+      collapse:
+        type: boolean
+        description: Signifies that the section should initially be displayed in a collapsed state
```

Comment:
I've added a boolean in to cater for `accordion` sections. We discussed previously that these are actually just sections added to the end of the other sections. Is this suitable or should it be renamed or more detail added?

Reply:
I think this is ok, but should possibly be `collapsable` or something as `collapse` implies that it should be collapsed to begin with, which in turn implies that the others are collapsable but start expanded. At least this is how I see it.

Reply:
I'm going with this, but using the spelling `collapsible`. It was hard to find a definitive answer on …able/…ible but the latter is in the dictionary. https://dictionary.cambridge.org/dictionary/english/collapsible